src/formatters/CheckboxFormatter.tsx

```diff
@@ -28,8 +28,10 @@ const checkbox = css`
   block-size: 20px;
   border: 2px solid var(--rdg-border-color);
   background-color: var(--rdg-background-color);
-  .${checkboxInput}:checked + & {
+  .${checkboxInput}:checked:not(:disabled) + & {
```

Comment:
```suggestion
  .${checkboxInput}:checked:enabled + & {
```

Reply:
I inverted the changes in the last commit to use `:enabled` instead.